### ISSUE TITLE
Option to disable mod features on config

### ISSUE THREAD

**Issue by ffuentesm** [benbenlaw/opolisutilities#8]
I really enjoy using the Casting mod, but I find most of the Opolis Utilities a bit distracting/unwanted from my mod list. 
Is it possible to disable the mod on the config? To that mobs do not drop loot boxes, or the items do not show up on my crafting?

**Comment by benbenlaw** [comment]
yeah you can disable the loot bags in the config for opolis utilities, you can use kubejs to remove the recipes and hide the item in jei for stuff you don't want. In the future casting will not be dependant on utilities but at the moment it is
